File: app/src/main/java/knf/kuma/tv/details/ChaptersListPresenter.java
package knf.kuma.tv.details;

import androidx.leanback.widget.ListRowPresenter;
import androidx.leanback.widget.RowPresenter;

public class ChaptersListPresenter extends ListRowPresenter {

    private int position = 0;

    public ChaptersListPresenter(int position) {
        this.position = position;
    }

    @Override
    protected void onBindRowViewHolder(RowPresenter.ViewHolder holder, Object item) {
        super.onBindRowViewHolder(holder, item);
        ViewHolder vh = (ListRowPresenter.ViewHolder) holder;
        vh.getGridView().setSelectedPosition(position);
    }
}

File: app/src/main/java/knf/kuma/tv/details/ChaptersListPresenter.kt
package knf.kuma.tv.details

import androidx.leanback.widget.ListRowPresenter
import androidx.leanback.widget.RowPresenter

class ChaptersListPresenter(val position: Int) : ListRowPresenter() {

    override fun onBindRowViewHolder(holder: RowPresenter.ViewHolder, item: Any) {
        super.onBindRowViewHolder(holder, item)
        val vh = holder as ViewHolder
        vh.gridView.selectedPosition = position
    }
}
